### DISCUSSION TITLE
Redirection

### DISCUSSION THREAD

**Discussion by j-clavoie** [tcort/link-check#39] [Ideas]
Hello, I would like to know if it's possible to know that a link has been redirected. 
I did some tests with redirected links. I got a status 200 (link is working well) but I don't find a way to know that the link has been redirected and where it has been redirected (new URL).

Is it possible  to add properties in the "LinkCheckResult" method to indicate that a link has been redirected?
Something like: 
```
this.redirected = BOOLEAN value
this.redirectedLink = STRING Value that contains the redirection URL
```
**Or** let me know how I can know that URL has bee redirected?

Thanks.

**Comment by NicolasMassart** [comment]
Not yet, we are looking at a way to do that because it's something also asked by Markdown link check users.
See for instance https://github.com/tcort/markdown-link-check/issues/151
So yes there will be things for that. We need to define it clearly.

**Comment by Jsan2020** [comment]
Love `link-check` and wish to use it to check for strictly "200" response codes. I tried adding `{ aliveStatusCodes: [200] }` but redirects are followed by default. Is there any way to treat redirects as errors?
